scripts/liminal

```diff
@@ -105,17 +108,15 @@ def deploy_liminal_apps(path, clean):
         shutil.copyfile(config_file, target_yml_name)
 
 def is_airflow_running():
```

Comment:
please rename to liminal_is_running (since you are checking for liminal, not airflow)